tomcat/src/main/java/org/apache/coyote/response/HttpResponseGenerator.java

```diff
@@ -0,0 +1,18 @@
+package org.apache.coyote.response;
+
+public class HttpResponseGenerator {
+    public static String getOkResponse(String mimeType, String responseBody) {
+        return String.join("\r\n",
+                "HTTP/1.1 200 OK ",
+                "Content-Type: " + mimeType + ";charset=utf-8 ",
+                "Content-Length: " + responseBody.getBytes().length + " ",
+                "",
+                responseBody);
+    }
+
+    public static String getFoundResponse(String url) {
+        return String.join("\r\n",
+                "HTTP/1.1 302 FOUND ",
+                "Location:  " + url);
+    }
```

Comment:
리다이렉트 하는 곳들에서 `HttpResponseGenerator.getFoundResponse("http://localhost:8080/index.html")` 이 함수가 반복되고 있어요. 지금은 우리 어플리케이션 안에서 리다이렉트만 할테니, 도메인과 포트는 함수 안에서 고정해두는게 어떨까요?
```suggestion
    public static String getFoundResponse(String path) {
        return String.join("\r\n",
                "HTTP/1.1 302 FOUND ",
                "Location: http://localhost:8080" + path);
    }
```